soh/src/code/z_parameter.c

```diff
@@ -1415,13 +1415,18 @@ void Inventory_SwapAgeEquipment(void) {
         // When becoming adult, remove swordless flag since we'll get master sword
         // Only in rando to keep swordless link bugs in vanilla
         if (gSaveContext.n64ddFlag) {
+            if (!Randomizer_GetSettingValue(RSK_SHUFFLE_MASTER_SWORD)) {
```

Comment:
Maybe combine these if statements together and update the comment above regarding MS shuffle